.dockerignore

```diff
@@ -12,6 +12,7 @@
 !*.txt
 !*.cfg
 !**/requirements.txt
+!gcovr/version.py
```

Comment:
A few lines above there is a `!gcovr/**.py` pattern, so I think adding this ought to be redundant?

Reply:
You're right.